src/components/Select/index.js

```diff
@@ -106,6 +119,11 @@ export function Select({ items, floatingMenu, onChange, id, label, ...props }) {
             }}
             aria-selected={selectedItem === item ? 'true' : 'false'}
           >
+            {flag && (
+              <StyledFlagsContainer>
+                <Flag countryCode={item ? item.value : null} />
+              </StyledFlagsContainer>
+            )}
```

Comment:
I see you added the flag as `leftAdornment`-prop above, so I think this block can be removed?

Reply:
Actually I think all references to `flag` can be removed from the `<Select />` component.